MIGRATION.md

```diff
@@ -1,5 +1,8 @@
 # Migration guides
 
+## 3.8.x to 3.9.0
+- The `Radar.autocomplete(query:near:layers:limit:country:expandUnits:completionHandler:)` method is now `Radar.autocomplete(query:near:layers:limit:country:mailable:completionHandler:)`.
```

Comment:
Call out explicitly that `expandUnits` is deprecated, and that the API now always behaves as if `expandUnits` true.